index.html

```diff
@@ -1885,6 +1887,40 @@ <h2>
           shipping option. <a>User agents</a> MAY indicate pending fields in
           the user interface for the payment request.
         </dd>
+        <dt>
+          <dfn>type</dfn> member
```

Comment:
Since there is no type for the "normal" payment items, I wonder if this should be named "specialType" or something? It's probably fine as just `type` though.

Reply:
I'd like to just keep it as `type`.